logging/callback.go

```diff
@@ -0,0 +1,68 @@
+package logging
+
+// CallbackLogger is an implementation of Logger that forwards calls to Log and
```

Comment:
```suggestion
// CallbackLogger is an implementation of Logger that forwards log messages to user-supplied callback functions.
```

I can't do multi-line suggestions in the PR interface, so could you please apply word-wrapping to these suggestion.